torchgeo/datasets/l8biome.py

```diff
@@ -0,0 +1,280 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8 Biome dataset."""
+
+import glob
+import os
+import re
+from typing import Any, Callable, Dict, List, Optional, Sequence, cast
+
+import matplotlib.pyplot as plt
+import torch
+from rasterio.crs import CRS
+from torch import Tensor
+
+from .geo import RasterDataset
+from .utils import BoundingBox, download_url, extract_archive
+
+
+class L8Biome(RasterDataset):
+    """L8 Biome dataset.
+
+    The `L8 Biome <https://landsat.usgs.gov/landsat-8-cloud-cover-assessment-validation-data>`__ dataset
+    is a cloud validation dataset of Pre-Collection Landsat 8
+    Operational Land Imager (OLI) Thermal Infrared Sensor (TIRS)
+    terrain-corrected (Level-1T) scenes.
+
+    Dataset features:
+
+    * images evenly divided between eight unique biomes
+    * 5 cloud cover categories
+
+    Dataset format:
+
+    * Images are stored as GeoTIFF files corresponding to different bands
+    * Each cloud mask is in ENVI binary format.
+
+    Dataset classes:
+
+    0. Fill
+    1. Cloud Shadow
+    2. Clear
+    3. Thin Cloud
+    4. Cloud
+
+    If you use this dataset in your research, please cite the following:
+
+    * https://doi.org/10.5066/F7251GDH
+    * https://doi.org/10.1016/j.rse.2017.03.026
+
+    .. versionadded:: 0.5
+    """  # noqa: E501
+
+    url = "https://huggingface.co/datasets/torchgeo/l8biome/blob/main/{}.tar.gz"  # noqa: E501
```

Comment:
```suggestion
    url = "https://huggingface.co/datasets/torchgeo/l8biome/resolve/main/{}.tar.gz"  # noqa: E501
```
You're still downloading the HTML file instead of the raw data.

P.S. If you click "commit changes" you don't have to worry about copying the changes correctly, it will add my suggested change directly to the PR.